blocks/bricks/base.py

```diff
@@ -556,6 +582,7 @@ def __init__(self, name=None):
         self.name = name
 
         self.children = []
```

Comment:
Not sure I get it. You use `_children` it the `children` property, and here you create another `children` attribute. And also you want `children` property always return `Children` object. Seems like there is some mismatch here.

Maybe you want `self._chidren = Children(self, [])`?